themes-default/slim/static/js/store.js

```diff
@@ -338,6 +440,14 @@ const store = new Puex({
                 });
             });
         },
+        setConfig(context, { section, config }) {
+            const { dispatch } = context;
+
+            if (section !== 'main') {
+                return;
+            }
+            return api.patch('config/' + section, config).then(setTimeout(() => dispatch('getConfig'), 500));
+        },
         getShow(context, { indexer, id }) {
             return api.get('/series/' + indexer + id).then(res => {
                 store.commit(ADD_SHOW, res.data);
```

Comment:
Some actions are using `store.commit` and some `context.commit`.
Can we stick to one or the other?
I don't understand the difference, which means I'm bound to use the incorrect one.